src/sql/src/parsers/create_parser.rs

```diff
@@ -977,6 +1064,43 @@ ENGINE=mito";
 
         let result4 = ParserContext::create_with_dialect(sql4, &GenericDialect {});
         assert!(result4.is_err());
+
+        let sql = r"
+CREATE TABLE monitor (
+  host_id    INT,
+  idc        STRING,
+  ts         TIMESTAMP TIME INDEX DEFAULT CURRENT_TIMESTAMP,
+  cpu        DOUBLE DEFAULT 0,
+  memory     DOUBLE,
+  TIME INDEX (ts),
```

Comment:
Should we test the case that the column marked as "TIME INDEX" in its options is not the same column defined in "TIME INDEX()"?

Reply:
It's semantic checking, not the responsibility of the parser.

Reply:
But I think we can check it in the parser too, I'll add a validation.